hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```diff
@@ -79,9 +80,8 @@ public Data apply(Object input) {
         }
     };
     protected final ConstructorFunction<Void, RecordFactory> recordFactoryConstructor;
-    protected final boolean memberNearCacheInvalidationEnabled;
     /**
-     * Holds number of registered {@link com.hazelcast.map.impl.nearcache.InvalidationListener} from clients.
+     * Holds number of registered {@link InvalidationListener} from clients.
```

Comment:
We should put this `com.hazelcast.map.impl.nearcache.InvalidationListener` import back into the JavaDoc, it's not used otherwise.